autoware_launch/launch/components/tier4_system_component.launch.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<launch>
+  <arg name="run_mode" default="online" description="options: online, planning_simulation"/>
+  <arg name="sensor_model" description="sensor model name"/>
+  <arg name="launch_system_monitor" default="true" description="launch system monitor"/>
+
+  <include file="$(find-pkg-share tier4_system_launch)/launch/system.launch.xml">
+
+    <arg name="run_mode" value="$(var run_mode)"/>.
+    <arg name="sensor_model" value="$(var sensor_model)"/>
+    <arg name="launch_system_monitor" value="$(var launch_system_monitor)"/>
+
+    <arg name="component_state_monitor_topic_path" value="$(find-pkg-share autoware_launch)/config/system/component_state_monitor/topics.yaml"/>
+    <arg name="emergency_handler_param_path" value="$(find-pkg-share autoware_launch)/config/system/emergency_handler/emergency_handler.param.yaml"/>
+    <arg name="mrm_comfortable_stop_operator_param_path" value="$(find-pkg-share autoware_launch)/config/system/mrm_comfortable_stop_operator/mrm_comfortable_stop_operator.param.yaml"/>
+    <arg name="mrm_emergency_stop_operator_param_path" value="$(find-pkg-share autoware_launch)/config/system/mrm_emergency_stop_operator/mrm_emergency_stop_operator.param.yaml"/>
+    <arg name="system_error_monitor_param_path" value="$(find-pkg-share autoware_launch)/config/system/system_error_monitor/system_error_monitor.param.yaml"/>
+    <arg name="system_error_monitor_planning_simulator_param_path" value="$(find-pkg-share autoware_launch)/config/system/system_error_monitor/system_error_monitor.planning_simulation.param.yaml"/>
+    <arg name="diagnostic_aggregator_system_param_path" value="$(find-pkg-share autoware_launch)/config/system/diagnostic_aggregator/system.param.yaml"/>
+    <arg name="diagnostic_aggregator_vehicle_param_path" value="$(find-pkg-share autoware_launch)/config/system/diagnostic_aggregator/vehicle.param.yaml"/>
```

Comment:
Minor question: Do you know why this diagnostic_aggregator parameter files are stored in a different structure from awf/autoware_launch?
- awf/autoware_launch: `config/system/system_error_monitor/diagnostic_aggregator/vehicle.param.yaml`
- tier4/autoware_launch: `config/system/diagnostic_aggregator/vehicle.param.yaml`

Considering the sync from awf/autoware_launch -> tier4/autoware_launch which will be introduced after this PR, we want to mitigate unnecessary diffs as much as possible (not necessarily in this PR, though)

Reply:
@kminoda This structural difference is not critical for the system_error_monitor behavior, and I asked @ito-san to confirm this structural difference is not essential.

Reply:
OK, thanks!